front-end/legeno-around-here/src/components/pages/post/CommentItem.js

```diff
@@ -54,7 +60,20 @@ const CommentItem = ({ comment, myId }) => {
           <Typography variant='subtitle1' color='textSecondary'>
             {comment.creator.nickname}
           </Typography>
-          <ListItemText primary={comment.writing} secondary={convertDateFormat(comment.createdAt)} />
+          <Typography variant='subtitle1'>
+            {comment.writing.split('\n').map((line) => {
+              keyValue += 1;
+              return (
+                <span key={keyValue}>
+                  {line}
+                  <br />
+                </span>
+              );
+            })}
```

Comment:
keyValue를 사용하지 않고 map 안의 두 번째 인자에 index를 넣으면 자동으로 증가시켜줍니다 :)

```suggestion
{comment.writing.split('\n').map((line, index) => {
              return (
                <span key={index}>
                  {line}
                  <br />
                </span>
              );
            })}
```

Reply:
별 생각 없이 복사해서 사용한 코드였는데,

index가 현재 값의 위치인가보네요!!! 와 감사합니당!!!